src/main/resources/inventory.avro

```diff
@@ -12,20 +12,22 @@
                     }
                 }},
                 {"name": "quantity", "type": {
-                    "type": "long",
+                    "type": "int",
```

Comment:
any reason for changing this to `int` from long?

Reply:
IMO if there is no requirement to change the type, then let's keep it as is

Reply:
Restoring to `long`, thanks @shaikzakiriitm